src/frontend/src/pages/login/login.tsx

```diff
@@ -1,7 +1,34 @@
 import React from 'react';
+import Grid from '@material-ui/core/Grid';
+
+// for fields
+import TextField from '@material-ui/core/TextField';
+
+// for submit
+import Button from '@material-ui/core/Button';
+import {
+    Link,
+} from "react-router-dom"
 
 export const Login: React.FC = () => {
     return (
-        <div>Login Page</div>
+        <div>
+            <Grid container spacing={3}>
+                <Grid item xs={12}>
+                    <h1>Login Page</h1>
```

Comment:
```suggestion
                    <Typography variant="h1">Login Page</Typography>
```
This abstracts the heading levels using Material-UI components so that you can use React properties in the framework instead of raw CSS to style the components.